src/Console/stubs/validator.stub

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace DummyNamespace;
+
+use Code16\Sharp\Form\Validator\SharpFormRequest;
+
+class DummyClass extends SharpFormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            //
+        ];
+    }
+
+    /**
+     * Configure the validator instance.
+     *
+     * @param  \Illuminate\Validation\Validator  $validator
+     * @return void
+     */
+    public function withValidator($validator)
```

Comment:
Not sure why you put this method bu default?

Reply:
That was in there because I needed if for my use-case when I originally wrote the generators but I agree, it shouldn't be in package. Have removed it.